coderd/database/modelmethods.go

```diff
@@ -1,9 +1,36 @@
 package database
 
 import (
+	"sort"
+
 	"github.com/coder/coder/coderd/rbac"
 )
 
+type AuditableGroup struct {
+	Group
+	Members []GroupMember `json:"members"`
+}
+
+func (g Group) Auditable(users []User) AuditableGroup {
```

Comment:
Comment this function

```suggestion
// Auditable returns an object that can be used in audit logs.
// Covers both group and group member changes.
func (g Group) Auditable(users []User) AuditableGroup {
```